packages/enketo-core/src/js/widget.js

```diff
@@ -187,6 +194,10 @@ class Widget {
         this.element.dispatchEvent(event.Change());
     }
 
+    async prepareData() {
+        return Promise.resolve();
+    }
```

Comment:
why not reuse any of "init"/"setup"/"before"/etc. words usually used as antonyms for the already existing "cleanup"?